### DISCUSSION TITLE
Separate lr for text encoder and unet

### DISCUSSION THREAD

**Discussion by giteeeeee** [d8ahazard/sd_dreambooth_extension#1217] [Ideas]
Text encoder is more sensitive to high learning rates than unet during training. Despite this, they use the same lr value.

The current solution is to limit the training steps of tenc to a percentage to that of unet, but shouldn't it make more sense to just use two separate lr values?

So is it technically possible to have a field where we can input the lr value just for tenc?

**Comment by morphinapg** [comment]
I agree. Lora does this, so it would make sense to do this for non-lora training too.
